src/views/connected/coordinateur/StructureDetails.js

```diff
@@ -160,7 +160,24 @@ function StructureDetails() {
                   </button>
                 </span>
               )}
-              {structure?.conseillers?.length === 0 &&
+              {structure?.conseillersRecruter?.length === 0 &&
+                <span className="fr-col-12">-</span>
+              }
+            </div>
+            <div className="fr-mb-3w fr-grid-row">
+              <strong>Profils valid&eacute;s</strong>
+              {structure?.conseillersValider?.map((conseiller, idx) =>
+                <span key={idx} className="fr-col-12" style={{ height: '2rem' }}>
+                  <button
+                    style={{ paddingLeft: '0' }}
+                    title="D&eacute;tail"
+                    className="fr-text--md"
+                    onClick={() => window.open(`/${roleActivated}/candidat/${conseiller?._id}`)}>
+                    {conseiller?.idPG}&nbsp;-&nbsp;{formatNomConseiller(conseiller)}
```

Comment:
Ca évitera d'avoir un undefined affiché le temps que ca arrive
```suggestion
                    {conseiller?.idPG}&nbsp;-&nbsp;{conseiller ? formatNomConseiller(conseiller) : ''}
```